sass/style.sass

```diff
@@ -82,12 +107,103 @@ h3
     margin-top: $px20
     font-size: 18px
 
+  &--send
+    @extend .portfolio-button--first
+    min-width: 20%
+    height: 50px
+    padding: 0 $px20
+    font-size: 18px
+    i
+      margin-right: $px20
+
+.block-wrapper
+  display: flex
+
+.form-wrapper
+
+.contact-items
+  padding-bottom: 40px
+  border-bottom: 1px solid $white
+
+.contact-footer
+  width: 100%
+  height: auto
+  color: $white
+  display: grid
+  grid-template-columns: 1fr 1fr 1fr
+  grid-gap: 10px
+  background-color: $grey-hard
+  padding: $px20
+
+  a
+    color: $white
+
+
+  .f-third-block
```

Comment:
I wouldn't name classes as `f-first-block`, `f-third-block` and `f-second-block`
- I'd use more meaningful names or use `nth-child` if it's enouth;
- you used them just to style tags. I'd rather style by 'tag' class. Because you can use tags in different places. And active (or first) tag can differs from others in another place too.

Reply:
@silchencko got it